.gitattributes

```diff
@@ -1,3 +1,3 @@
 *.ts text eol=lf
-*.js test eol=lf
+*.js text eol=lf
```

Comment:
reverted change from last PR

Reply:
I would separate this into a separate PR. There shouldn't be any conflicts with this feature branch.